LiskHQ/lisk-desktop
# Add webpack hot module replacement to the development flow - Closes #2733

### What issue have I solved?

Resolves #2733 

### How have I implemented/fixed it?

- updated the version of react-hot-loader
- Added react-hot-loader to the babel configuration
- Added react-hot-loader to App component


### How has this been tested?

1. run `npm run dev`
2. login with a passphrase
3. change a css or js file
4. the change that you made has to apply but you should stay login

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-desktop/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-desktop/blob/development/docs/CSS_GUIDE.md)
